apps/server/src/lib/mongoose/seeders/generate-reports.ts

```diff
@@ -80,21 +112,33 @@ function generateReport(
   const report: ReportModel = {
     _id: { targetId, cycleId },
     summary: faker.lorem.paragraph({ min: 2, max: 4 }),
-    status: faker.helpers.arrayElement([
-      'Nomination',
-      'Review',
-      'Report',
-      'Completed',
-    ]),
+    status: faker.helpers.objectValue(REPORT_STATUS),
     reviews: {
       peers: peerReviews,
-      self: generateReview(targetId, metricCount, maxRating),
-      manager: generateReview(managerId, metricCount, maxRating),
- 
+      self: generateReview(metricCount, maxRating, targetId),
+      manager: generateReview(metricCount, maxRating, managerId),
     },
   }
 
   return report
 }
 
-export default generateReport
+export function pickRandomReviewers(
```

Comment:
`pickRandomReviewers()` isn't new, it just used to live on the `seeder-random` file. Now it gets exported from here so both seeder files can use it.